assets/src/application/ui/layout/espressoDropdownMenu/style.scss

```diff
@@ -341,9 +341,10 @@
 	.components-popover__content {
 		box-shadow: var(--ee-box-shadow-small);
 		height: auto;
-		min-width: 200px;
+		min-width: 245px;
```

Comment:
this was changed because of this: 
![image](https://user-images.githubusercontent.com/1477453/77926388-bc76ce00-72ae-11ea-9ac7-926c0e11b594.png)

Reply:
we can try removing the `datetime`, like this:
![image](https://user-images.githubusercontent.com/1477453/77926711-1aa3b100-72af-11ea-9949-43de59f62b99.png)